pycipapi/models.py

```diff
@@ -210,10 +209,24 @@ def cancer_participant(self):
         if self.interpretation_request_data and self.sample_type == 'cancer':
             return self.interpretation_request_payload.cancerParticipant
 
+    @property
+    def members(self):
+        if self.interpretation_request_data and self.sample_type == 'raredisease':
+            return [participant.participantId for participant in self.pedigree.members if participant.samples]
```

Comment:
@MattWellie ,

Don't you mean `if participant. participantId ` here?

Reply:
Mmm, no. The participants all have an ID because It's mandatory in the model. The samples can be an empty list. This includes all the NR (not recruited) participants who are only available in the pedigree (probably useless to queries)